core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -44,6 +44,8 @@ final case class WriterT[F[_], L, V](run: F[(L, V)]) {
 
   def reset(implicit monoidL: Monoid[L], functorF: Functor[F]): WriterT[F, L, V] =
     mapWritten(_ => monoidL.empty)
+
+  def show(implicit F: Show[F[(L, V)]]): String = F.show(run)
```

Comment:
Yeah, the `Show` instance is delegating to this method.  I modeled this off the change for OptionT (see: https://github.com/typelevel/cats/pull/600/files; and in particular: https://github.com/typelevel/cats/blob/6dbd4b04aff336867f563c9f6ec04fde828aa816/core/src/main/scala/cats/data/OptionT.scala#L90).